README.md

```diff
@@ -20,16 +20,17 @@ Endpoints
 ----------
 
 - All callbacks are passed an error and response: `callback(err, res)`.
-- Please refer to Localize's [API Docs](https://help.localizejs.com/reference#api-overview) for endpoint details.
+- Please refer to Localize's [API Docs](https://help.localizejs.com/reference#api-overview) for the most up-to-date endpoint details.
 
 **localizeServices.project**
 
     localizeService.project.create({
             name: 'project1',
             sourceLanguage: 'en',
+            type: 'web'
         }, callback);
 
-    Required fields:  name, sourceLanguage       
+    Required fields:  name, sourceLanguage, type
```

Comment:
Type of project need not be mandatory. It defaults `web`. The document should also be updated. It marks type as mandatory, but the explanation reads `defaults to web if not specified.`  https://help.localizejs.com/reference/api-projects-create

Reply:
Also, should we make `activeLanguages` mandatory in the backend API? It is mandatory while creating projects through dashboard https://github.com/Localize/Localize/blob/master/services/dashboard/controllers/project.js#L44. But, not mandatory while it is created through rest API https://github.com/Localize/Localize/blob/master/services/api/controllers/project.js#L45-L49.

Reply:
I fixed your first comment and created a task for the second comment: https://shapesuite.atlassian.net/browse/PBI-1026